### PR TITLE
Complete B3 Migration of corehq.domain

### PR DESCRIPTION
**Pull request by biyeun**
Migrates some smaller views and updates missed b2 styles in templates. I don't think this needs QA as the views that were migrated are pretty straightforward / small. A good look through would be great tho

@orangejenny 
cc @proteusvacuum 

screen shot 2016-04-28 at 5 58 07 pm
screen shot 2016-04-28 at 5 55 55 pm
screen shot 2016-04-28 at 5 49 22 pm
screen shot 2016-04-28 at 4 59 48 pm
screen shot 2016-04-28 at 4 36 14 pm

### CODE REVIEW

File: custom/dhis2/forms.py [hunk 1]
```diff
@@ -28,6 +32,34 @@ class Dhis2SettingsForm(forms.Form):
                     'e.g. "Fermathe Clinic"'),
         required=False)
 
+
+
+    def __init__(self, *args, **kwargs):
```
Comment (dimagimon) [inline custom/dhis2/forms.py]: E303 too many blank lines (3)

File: corehq/apps/domain/templates/domain/admin/commtrack_action_table.html [hunk 2]
```diff
@@ -23,7 +23,7 @@
             <select class="form-control" data-bind="value: type, options: $parent.action_types, optionsText: 'label', optionsValue: 'value'"></select>
           </td>
           <td>
-            <button type="button" class="btn btn-danger" data-bind="click: $parent.remove_action"><i class="icon-trash icon-white"></i> {% trans "Remove" %}</button>
+            <button type="button" class="btn btn-danger" data-bind="click: $parent.remove_action"><i class="fa fa-remove"></i> {% trans "Remove" %}</button>
```
Comment (orangejenny) [inline corehq/apps/domain/templates/domain/admin/commtrack_action_table.html]: Elsewhere in this file, `"css: { 'error': keyword_error }"` should be `"css: { 'has-error': keyword_error }"` and `help-inline` can be removed (replace with `help-block`?).

File: corehq/apps/domain/templates/domain/admin/repeater_row.html [hunk 3]
```diff
@@ -13,7 +13,7 @@
             {% trans "View forwarded records" %}
         </a>
         <a class="btn btn-danger" href="#stop_forwarding_{{ repeater.get_id }}" data-toggle="modal"><i
-                class="icon icon-white icon-remove"></i> {% trans "Stop Forwarding" %}</a>
+                class="fa fa-remove"></i> {% trans "Stop Forwarding" %}</a>
```
Comment (orangejenny) [inline corehq/apps/domain/templates/domain/admin/repeater_row.html]: A couple lines down, the `modal-title` should be `h4`.

File: corehq/apps/domain/templates/domain/partials/project_version_display.html [hunk 4]
```diff
@@ -3,8 +3,8 @@
 <form class="form-horizontal" action="{% url "domain_clear_published" domain %}" method="POST">
 {% csrf_token %}
 <div class="row-fluid">
-    <div class="span4"><h3>This project has been published</h3></div>
```
Comment (orangejenny) [inline corehq/apps/domain/templates/domain/partials/project_version_display.html]: Translations missing in a couple places in this file.

File: corehq/apps/domain/templates/domain/snapshot_settings.html [hunk 5]
```diff
@@ -134,7 +134,7 @@ <h3 style="display: inline">This project has not been published</h3>
     </table>
 </div>
 {% if error_message %}
-<div class="alert alert-error">{{ error_message }}</div>
+<div class="alert alert-danger">{{ error_message }}</div>
```
Comment (orangejenny) [inline corehq/apps/domain/templates/domain/snapshot_settings.html]: This file is missing some translations.

**Comment by orangejenny** [comment]
Looks good.

> I don't think this needs QA as the views that were migrated are pretty straightforward / small.

Did you load these pages locally? If yes, I agree this can skip QA.

**Comment by biyeun** [comment]
@orangejenny responded. yes tested locally

**Pull request merged**